subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/teamcity/TeamcityBuildLinkAccessor.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.android.lint.teamcity
+
+import com.avito.kotlin.dsl.getMandatoryStringProperty
+import com.avito.utils.gradle.envArgs
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import org.gradle.api.Project
+
+interface TeamcityBuildLinkAccessor {
+
+    fun getBuildUrl(): HttpUrl
+
+    fun getLintArtifactUrl(): HttpUrl
+
+    class Impl(private val project: Project) : TeamcityBuildLinkAccessor {
```

Comment:
Prefer to see classes like this in common modules without dependencies on Gradle